.github/workflows/test.yml

```
@@ -11,6 +11,42 @@ jobs:
+        run: |
+          failed=0
+          for f in $(find files -name '*.sh' -type f); do
+            if ! bash -n "$f" 2>&1; then
+              echo "FAIL: $f"
+              failed=1
+            fi
+          done
+          exit $failed
+
+      - name: Validate JSON files
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2044:warning:2:10: For loops over find output are fragile. Use find -exec or a while read loop

(shellcheck)